<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="2G53N13" failures="1" tests="1" name="Failure Test" time="0.002" errors="0" timestamp="2022-12-07T15:24:50 IST">
  <testcase classname="testcases.TestFailure" name="doLogin" time="0.001">
    <failure type="java.lang.AssertionError" message="Failing the login test">
      <![CDATA[java.lang.AssertionError: Failing the login test
at testcases.TestFailure.doLogin(TestFailure.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
</testsuite> <!-- Failure Test -->
